File: RailgunVB/Core/Utilities/EvalUtils.vb
Imports Discord
Imports Discord.Commands
Imports Discord.WebSocket
Imports TreeDiagram

Namespace Core.Utilities
    
    Public Class EvalUtils
    
        Public ReadOnly Client As DiscordShardedClient
        Public ReadOnly IClient As IDiscordClient
        Public ReadOnly Context As ICommandContext
        Public ReadOnly DbContext As TreeDiagramContext

        Public Sub New(client As DiscordShardedClient, iClient As IDiscordClient, context As ICommandContext, 
                       dbContext As TreeDiagramContext)
            Client = client
            IClient = iClient
            Context = context
            DbContext = dbContext
        End Sub
        
    End Class
    
End NameSpace

File: Railgun/Core/Utilities/EvalUtils.cs
using Discord;
using Discord.Commands;
using Discord.WebSocket;
using TreeDiagram;

namespace Railgun.Core.Utilities
{
    public class EvalUtils
    {
        public readonly DiscordShardedClient Client;
        public readonly IDiscordClient IClient;
        public readonly ICommandContext Context;
        public readonly TreeDiagramContext DbContext;

        public EvalUtils(DiscordShardedClient client, ICommandContext context, TreeDiagramContext dbContext) {
            Client = client;
            IClient = client;
            Context = context;
            DbContext = dbContext;
        }
    }
}